src/com/sap/piper/CommonPipelineEnvironment.groovy

```diff
@@ -0,0 +1,154 @@
+package com.sap.piper;
+
+import com.sap.piper.analytics.InfluxData
+
+public class CommonPipelineEnvironment {
```

Comment:
no need for `implements Serializable` ?

Reply:
I'm late to the party, since @radsoulbeard already merged the PR, but yes, that is reasonable. I never encountered an issue, but it should we serializable, since the CPS pattern relies on frequent serialization. I will open a follow up for that